content/07.pathogenesis.md

```diff
@@ -99,6 +99,17 @@ The dsRNA genome is transcribed and replicated to create viral mRNAs and new ssR
 From there, the virus can spread into other cells.
 In this way, the genome of SARS-CoV-2 provides insight into the pathogenic behavior of the virus.
 
+Evidence also suggests that SARS-CoV-2 may take advantage of the specific structure of endothelial cells to enter the circulatory system.
+Endothelial cells are specialized epithelial cells [@doi:10.1055/s-0030-1253446] that facilitate exchange between the blood and the tissue of the circulatory system [@PMID:362947].
+The endothelium is lined with glycocalyx, which is a soluble network made up of proteins and carbohydrates, primarily glycans and glycoconjugates [@doi:10.1007/s00424-007-0212-8; @doi:10.1016/j.cell.2020.09.033].
+An analysis of the interactions between SARS-CoV-2 and the glycocalyx revealed that the SARS-CoV-2 spike protein can utilize heparan sulfate (HS), a polysaccharide that attaches to proteoglycans within the glycocalyx, to assist in binding to endothelial cells [@doi:10.1016/j.cell.2020.09.033].
+This study identified a HS binding site near the ACE2 binding site on the viral RBD, and modeling suggested that HS binding yields an open conformation that facilitates binding to ACE2 on the cell surface [@doi:10.1016/j.cell.2020.09.033].
+Degrading or removing HS was associated with decreased binding [@doi:10.1016/j.cell.2020.09.033].
+HS may also interact with the S1/S2 proteolytic cleavage site and other binding sites to promote binding affinity [@doi:10.1101/2020.05.10.087288].
+Additionally, the SARS-CoV-2 RBD was observed to bind to heparin [@doi:10.1016/j.cell.2020.09.033], another polysaccharide similar in structure to HS that is often administered as an anti-coagulant [@doi:10.1007/978-3-642-23056-1_8].
+This finding is particularly interesting because degradation of HS in the glycocalyx has previously been identified as an important contributor to ARDS and sepsis [@doi:10.1016/bs.ctm.2018.08.005], two common and severe outcomes of COVID-19, and suggests that HS could be a target for pharmaceutical inhibition of cell entry by SARS-CoV-2 [@doi:10.1038/s41421-020-00222-5; @doi:10.1016/j.ebiom.2020.102969; @doi:10.1016/j.amjcard.2020.08.005; @doi:10.3390/nu13020328; @doi:/10.3389/fimmu.2021.607306].
```

Comment:
```suggestion
This finding is particularly interesting because degradation of heparan sulfate in the glycocalyx has previously been identified as an important contributor to ARDS and sepsis [@doi:10.1016/bs.ctm.2018.08.005], two common and severe outcomes of COVID-19, and suggests that heparan sulfate could be a target for pharmaceutical inhibition of cell entry by SARS-CoV-2 [@doi:10.1038/s41421-020-00222-5; @doi:10.1016/j.ebiom.2020.102969; @doi:10.1016/j.amjcard.2020.08.005; @doi:10.3390/nu13020328; @doi:/10.3389/fimmu.2021.607306].
```